src/jit/codegencommon.cpp

```diff
@@ -180,6 +180,9 @@ CodeGen::CodeGen(Compiler* theCompiler) : CodeGenInterface(theCompiler)
     /* Assume that we not fully interruptible */
 
     genInterruptible = false;
+#if defined(_TARGET_ARM_) || defined(_TARGET_ARM64_)
```

Comment:
In the JIT, only, but not the VM, you can replace all occurrences of `defined(_TARGET_ARM_) || defined(_TARGET_ARM64_)` with `defined(_TARGET_ARMARCH_)`.

Reply:
Changed